[2022-02-28 07:23:40,835] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: test_s3_connection.upload_file_to_S3 scheduled__2022-02-25T00:00:00+00:00 [queued]>
[2022-02-28 07:23:40,847] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: test_s3_connection.upload_file_to_S3 scheduled__2022-02-25T00:00:00+00:00 [queued]>
[2022-02-28 07:23:40,849] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 07:23:40,849] {taskinstance.py:1244} INFO - Starting attempt 9 of 9
[2022-02-28 07:23:40,850] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 07:23:40,864] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): upload_file_to_S3> on 2022-02-25 00:00:00+00:00
[2022-02-28 07:23:40,869] {standard_task_runner.py:52} INFO - Started process 14458 to run task
[2022-02-28 07:23:40,878] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_s3_connection', 'upload_file_to_S3', 'scheduled__2022-02-25T00:00:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/example_dag.py', '--cfg-path', '/tmp/tmpvxuna1iu', '--error-file', '/tmp/tmphgjkkirx']
[2022-02-28 07:23:40,881] {standard_task_runner.py:77} INFO - Job 58: Subtask upload_file_to_S3
[2022-02-28 07:23:40,982] {logging_mixin.py:109} INFO - Running <TaskInstance: test_s3_connection.upload_file_to_S3 scheduled__2022-02-25T00:00:00+00:00 [running]> on host cdd2ec182991
[2022-02-28 07:23:41,055] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_s3_connection
AIRFLOW_CTX_TASK_ID=upload_file_to_S3
AIRFLOW_CTX_EXECUTION_DATE=2022-02-25T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-25T00:00:00+00:00
[2022-02-28 07:23:41,067] {base_aws.py:194} INFO - Credentials retrieved from extra_config
[2022-02-28 07:23:42,009] {python.py:175} INFO - Done. Returned value was: None
[2022-02-28 07:23:42,024] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=test_s3_connection, task_id=upload_file_to_S3, execution_date=20220225T000000, start_date=20220228T072340, end_date=20220228T072342
[2022-02-28 07:23:42,077] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-28 07:23:42,120] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
